backend/app.py

```diff
@@ -0,0 +1,94 @@
+import os
+from dotenv import load_dotenv
+from flask import Flask, jsonify, request, render_template, url_for,redirect,flash
+from flask_sqlalchemy import SQLAlchemy
+
+# .env
+load_dotenv()
+
+app = Flask(__name__)
+
+# database connection
+url = os.getenv("DATABSE_URL") # variable from .env
+app.config["SQLALCHEMY_DATABASE_URI"] = url
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+# print(url)
+
+
+
+# DATABASE MODELS FOR CONSISTENT DATA STRUCTURE:
+
+# Use a class to define a table structure.
+# Incoming data will be required to fit the class for consistency.
+# When the backend gets run, any non-existing tables 
+# that have a newly defined class will be (*)should be)
+# automatically created in the database.
+
+
+
+
+# sensors model - for sensor metadata
+class sensors(db.Model):
+    _id = db.Column("id", db.Integer, primary_key=True)
+    sampling = db.Column(db.Integer)
+    floor = db.Column(db.String(255))
+    user = db.Column(db.String(255))
+
+
+
+
+# ENDPOINTS:
+
+# Only existing endpoint right now receives the sensorid,
+# the sampling rate of the sensor, the floor type, and 
+# the user's name.
+
+# This endpoint is for functionality testing purposes.
+# This endpoint works with local deployment.
+
+
+# retrieve sensor metadata
+@app.route('/api/sensor_metadata', methods=['POST'])
+def add_data():
+    data = request.json
+
+    try:
+        new_data = sensors(
+            _id=data['sensorid'],
+            sampling=data['sampling'],
+            floor=data['floor'],
+            user=data['user']
+        )
+
+        db.session.add(new_data)
+        db.session.commit()
+
+        return jsonify({"message": "Data added successfully"}), 201
```

Comment:
These http codes shouldn't be magic constants, they should be defined as a constant/variable. i.e. `HTTP_CREATED = 201`